src/components/TopBar/CurrentTask.js

```diff
@@ -14,6 +14,8 @@ export const CurrentTask = observer(({ store }) => {
   const [visibleComments, setVisibleComments] = useState(0);
 
   useEffect(() => {
+    store.commentStore.setAddedCommentThisSession(false);
```

Comment:
Does this work without newUI enabled? I'm worried we trade one problem for the other in the interim.

Reply:
Checked it, it is fine.